### DISCUSSION TITLE
MediaWiki import on 3.0

### DISCUSSION THREAD

**Discussion by jokay** [requarks/wiki#6481] [Help / Questions] [answered]
Hellu

Given the release of **3.0** is near ([ETA H2 2023 ](https://docs.requarks.io/releases)) we wanted to test some functionalities like the MediaWiki import but I can't find it on the latest main build.

When clicking on `Utilities` / `Import` / `Proceed` nothing happens, not sure if this is supposed to to anything already?

Any help would be appreciated.

**Comment by NGPixel** [comment, marked as answer]
There's no MediaWiki import planned for the near future. The import feature in Utilities will be for Wiki.js itself (from older versions or other installations), not from 3rd party products.

**Reply by jokay** [reply]
Thank you for the clarification.
